api/models.py

```diff
@@ -231,6 +231,9 @@ def save(self, *args: Any, **kwargs: Any) -> None:
 
 
 class Transcription(models.Model):
+    class Meta:
+        indexes = [models.Index(fields=["author", "submission", "original_id", "url"])]
```

Comment:
Do you mean to create an index that's the combination of these fields? I'm not sure if that's helpful as much as, for example, indexing the `original_id` or the `url` fields on their own. Perhaps I'm misreading how Django interacts with the database, though.

Reply:
...I don't know. Let me look into it.

Reply:
I think you're right. Will tweak.

Reply:
In other words, I'm not sure you intended to create a [composite index](https://en.wikipedia.org/wiki/Database_index#Column_order) of these fields. If you did, I'm curious how that would improve search speed/efficiency.